src/hb-aat-layout-lcar-table.hh

```diff
@@ -123,6 +123,20 @@ struct lcar
 			       unsigned int   *caret_count /* IN/OUT */,
 			       hb_position_t  *caret_array /* OUT */) const
   {
+    /* AAT::Lookup has no other way to detect whether it is returned from
+       a real and sanitized font data or from a null pool, this checks if
+       the table has been recognized valid by sanitizer by checking
+       table's major version which is zero if returned from a null pool and
+       non-zero if is from a sanitized font data, it is expected the other
+       calls of the table (unlikely to have more calls however) also do a
+       similar version check before calling the lookups used on the table */
+    if (!version.major)
+    {
+      if (caret_count)
+        *caret_count = 0;
+      return 0;
+    }
+
```

Comment:
This is wrong.  Why do u need to know that?  Just add a `has_data()` method and call that.  No comment needed.

Reply:
> Why do u need to know that? 

As explained I see no other way to check whether lookup is returned from a null pool or not IIUC